msgid ""
msgstr ""
"Project-Id-Version: oVirt Client\n"
"POT-Creation-Date: 2016-11-02 19:31+0000\n"
"PO-Revision-Date: 2016-11-02 19:32+0000\n"
"Last-Translator: Nicolás Kovac Neumann <nicolas@devels.es>\n"
"Language-Team: nicolas@devels.es\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.8.7.1\n"
"X-Poedit-KeywordsList: _;gettext;gettext_noop\n"
"X-Poedit-Basepath: .\n"
"X-Poedit-SearchPath-0: /home/nico/ovirt_client/ovirt_client\n"

#: /home/nico/ovirt_client/ovirt_client/about.py:51
#: /home/nico/ovirt_client/ovirt_client/credentials.py:90
#: /home/nico/ovirt_client/ovirt_client/credentials.py:94
#: /home/nico/ovirt_client/ovirt_client/credentials.py:164
#: /home/nico/ovirt_client/ovirt_client/credentials.py:178
#: /home/nico/ovirt_client/ovirt_client/credentials.py:260
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:268
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:271
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:277
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:283
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:285
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:289
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:291
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:359
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:399
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:416
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:420
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:441
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:446
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:449
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:451
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:484
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:489
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:630
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:844
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:891
msgid "apptitle"
msgstr "Cliente oVirt de escritorio"

#: /home/nico/ovirt_client/ovirt_client/about.py:53
msgid "appdesc"
msgstr "Un cliente simple de escritorio para oVirt"

#: /home/nico/ovirt_client/ovirt_client/about.py:56
msgid "written_by"
msgstr "Escrito por"

#: /home/nico/ovirt_client/ovirt_client/about.py:58
msgid "unofficial_project"
msgstr "Este proyecto NO es oficial."

#: /home/nico/ovirt_client/ovirt_client/about.py:63
#: /home/nico/ovirt_client/ovirt_client/credentials.py:228
msgid "ok"
msgstr "OK"

#: /home/nico/ovirt_client/ovirt_client/about.py:84
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:150
msgid "about"
msgstr "Información"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:63
msgid "loading"
msgstr "Cargando..."

#: /home/nico/ovirt_client/ovirt_client/credentials.py:79
msgid "authenticating"
msgstr "Autenticando..."

#: /home/nico/ovirt_client/ovirt_client/credentials.py:87
msgid "authenticated_and_storing"
msgstr "Autenticado, almacenando credenciales..."

#: /home/nico/ovirt_client/ovirt_client/credentials.py:90
#: /home/nico/ovirt_client/ovirt_client/credentials.py:94
#: /home/nico/ovirt_client/ovirt_client/credentials.py:164
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:277
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:359
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:399
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:420
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:446
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:449
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:630
msgid "error"
msgstr "Algo malo pasó..."

#: /home/nico/ovirt_client/ovirt_client/credentials.py:90
msgid "ovirt_connection_error"
msgstr "Error de conexión a oVirt"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:91
#: /home/nico/ovirt_client/ovirt_client/credentials.py:95
msgid "error_while_authenticating"
msgstr "Error autenticando"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:94
msgid "ovirt_request_error"
msgstr "Error de petición a oVirt"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:106
msgid "storing_credentials"
msgstr "Almacenando credenciales"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:112
msgid "successfully_authenticated"
msgstr "Autenticación correcta"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:164
msgid "no_empty_creds"
msgstr ""
"Ni el nombre ni la contraseña deben ser vacíos. Por favor, introduzca sus "
"credenciales de oVirt."

#: /home/nico/ovirt_client/ovirt_client/credentials.py:178
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:271
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:484
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:844
msgid "confirm"
msgstr "Confirmar"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:178
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:844
msgid "confirm_quit"
msgstr "¿Está seguro de querer salir?"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:212
msgid "username"
msgstr "Nombre de usuario"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:214
msgid "password"
msgstr "Contraseña"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:225
msgid "remember_username_and_password"
msgstr "Recordar usuario y contraseña"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:231
msgid "cancel"
msgstr "Cancelar"

#: /home/nico/ovirt_client/ovirt_client/credentials.py:260
msgid "credentials"
msgstr "Credenciales"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:87
msgid "no_vms"
msgstr ""
"En este momento no dispone de permisos sobre máquinas en esta "
"infraestructura oVirt"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:138
msgid "refresh"
msgstr "Recargar"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:143
msgid "forget_credentials"
msgstr "Olvidar credenciales"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:155
msgid "exit"
msgstr "Salir"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:206
msgid "up"
msgstr "encendida"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:208
msgid "down"
msgstr "apagada"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:210
msgid "powering_down"
msgstr "apagando"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:212
msgid "wait_for_launch"
msgstr "encendiendo"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:214
msgid "powering_up"
msgstr "encendiendo"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:216
msgid "rebooting"
msgstr "reiniciando"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:230
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:247
msgid "shut_down"
msgstr "apagar"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:232
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:249
msgid "power_on"
msgstr "encender"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:253
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:272
msgid "force_off"
msgstr "forzar fuera"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:244
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:271
msgid "current_vm_status"
msgstr "El estado actual de la máquina es"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:247
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:249
msgid "click_to_action"
msgstr "Haga click para"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:268
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:285
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:291
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:416
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:451
msgid "warning"
msgstr "Aviso"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:268
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:285
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:291
msgid "vm_in_unchangeable_status"
msgstr ""
"No se puede cambiar el estado de una máquina mientras ya se esté realizando "
"una tarea de encendido/apagado"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:271
msgid "confirm_vm_status_change"
msgstr "Por favor, confirme el cambio del estado de la máquina virtual a"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:293
msgid "confirm_vm_force_off"
msgstr "Por favor, confirme que desea apagar por la fuerza la máquina virtual"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:277
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:446
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:630
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:757
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:769
msgid "unexpected_connection_drop"
msgstr "La conexión se ha perdido inesperadamente. La aplicación se cerrará."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:283
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:289
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:489
msgid "success"
msgstr "Correcto"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:283
msgid "shutting_down_vm"
msgstr "Apagando máquina..."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:289
msgid "vm_not_powered_up"
msgstr "Máquina está apagado"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:347
msgid "powering_up_vm"
msgstr "Encendiendo máquina..."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:305
#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:355
msgid "force_off_vm"
msgstr "Forzando la máquina ..."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:359
msgid "unexpected_request_error"
msgstr "Error inesperado al realizar una petición de consola a oVirt"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:359
msgid "check_vm_config_updated"
msgstr ""
"Por favor, verifique que los cambios de configuración de la máquina estén "
"aplicados y si es necesario, apague y encienda de nuevo la máquina"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:399
msgid "no_viewer_file"
msgstr ""
"No se pudo obtener el fichero de configuración del visor. Por favor, "
"inténtelo de nuevo."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:416
msgid "cannot_connect_if_vm_not_up"
msgstr ""
"No se puede iniciar el visor de una máquina si su estado no es encendida. "
"Asegúrese de que la máquina esté encendida primero."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:420
msgid "cannot_open_more_viewer_sessions"
msgstr ""
"Ya existe una sesión abierta para está máquina. No se puede abrir más de una "
"sesión a la vez."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:441
msgid "info"
msgstr "Información"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:441
msgid "acquiring_vm_from_pool"
msgstr ""
"Asignando una máquina del pool. Por favor, sea paciente. Una vez la máquina "
"esté disponible, automáticamente aparecerá en su panel."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:449
msgid "cannot_attach_vm_to_user"
msgstr ""
"No se le puede asignar la máquina. Bien usted ya posee el número máximo de "
"máquinas asignables, o no quedan más máquinas en el pool."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:451
msgid "object_is_not_a_vmpool"
msgstr "El objeto seleccionado no es un pool de máquinas."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:484
msgid "confirm_forget_creds"
msgstr "¿Está seguro de querer olvidar sus credenciales guardadas?"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:489
msgid "creds_forgotten"
msgstr "Credenciales olvidadas correctamente."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:507
msgid "vmpool"
msgstr "Pool de máquinas"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:515
msgid "grab_vm_vmpool"
msgstr "Asignarse una máquina del pool"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:564
msgid "connect"
msgstr "Conectar a la máquina virtual"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:567
msgid "viewer_already_opened"
msgstr "El visor se encuentra abierto para esta máquina"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:647
msgid "total_machines"
msgstr "Número de VMs"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:647
msgid "total_vmpools"
msgstr "Número de VmPools"

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:689
msgid "auto_logout_warn"
msgstr ""
"Si no acepta esta ventana, su sesión será cerrada en un espacio corto de "
"tiempo debido a inactividad."

#: /home/nico/ovirt_client/ovirt_client/ovirtclient.py:690
msgid "auto_logout_warn_title"
msgstr "Cliente oVirt de escritorio: Cierre de sesión forzado"
